identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeUtil.kt

```diff
@@ -26,10 +26,17 @@ internal object FunkeUtil {
     const val TAG = "FunkeUtil"
 
     const val EU_PID_MDOC_DOCTYPE = "eu.europa.ec.eudi.pid.1"
-    const val SD_JWT_VCT = "urn:eu.europa.ec.eudi:pid:1"
+
+    // Was changed recently, it seems
+    // TODO: read this from openid4vci server metadata (which we are yet to read).
+    const val SD_JWT_VCT = "https://example.bmi.bund.de/credential/pid/1.0"
 
     const val USE_AUSWEIS_SDK = true
 
+    // Determines if key attestation or of proof of possession should be used.
+    // TODO: decide this based on the openid4vci server metadata (which we are yet to read).
```

Comment:
You are right! I'll have another PR to address this (and actually read the metadata). Unfortunately this specific setting is non-standard (actually, IMHO, the way demo.pid-issuer.bundesdruckerei.de implemented it, it abuses the standard), so there either should be an extension or they should tweak the protocol. The only thing we can do right this moment is to have this semi-hardcoded. But this will all be there in the next PR.